packages/terra-button-group/tests/wdio/button-group-spec.js

```diff
@@ -1,61 +1,59 @@
-const viewports = Terra.viewports('tiny', 'huge');
-
-describe('Button Group', () => {
+Terra.describeViewports('Button Group', ['huge'], () => {
   describe('Text Button', () => {
     before(() => {
       browser.url('/#/raw/tests/terra-button-group/button-group/button-group-text');
       browser.moveToObject('#root', 0, 900); // move cursor to remove focus styles
     });
-    Terra.it.isAccessible();
-    Terra.it.matchesScreenshot({ selector: '#button-group-text' });
+
+    Terra.it.validatesElement({ selector: '#button-group-text' });
   });
 
-  describe('Button Group Wrapping', () => {
+  Terra.describeViewports('Button Group Wrapping', ['tiny', 'huge'], () => {
```

Comment:
This should still be a Mocha `describe`

Reply:
You think all the tests should be ran against tiny and not just these select few? Previously it was only these two groups of tests ran against tiny.

Reply:
I'm assuming Terra.describeViewports should never be nested, if you want to test different viewports for the same spec file they need to be split out as siblings.

```jsx
Terra.describeViewports('', () => {
});

Terra.describeViewports('', () => {
})
```

Reply:
Fixed in 516460b5681319749429cae1b796e1ffa03f0b1f